<launch>

  <param name="use_sim_time" value="false" />

  <!-- 打开地图 -->
  <node name="map_server" pkg="map_server" type="map_server" args="$(find bitathome_navigation_control)/maps/big.yaml" />

  <!-- 打开sick 定义base和laser的tf -->
  <include file = "$(find bitathome_navigation_control)/launch/navigation_configuration.launch" />

  <!-- 打开激光定位 提供码盘数据 -->
  <node pkg="laser_scan_matcher" type="laser_scan_matcher_node" name="laser_scan_matcher_node" output="screen">
    <param name="fixed_frame" value="odom" />
  </node>

  <!-- 打开底座控制 输出串口打开关闭信息 -->
  <node pkg="bitathome_hardware_control" name="hc_motor_cmd" type="hc_motor_cmd.py" output="screen" />
  
  <!-- 导入amcl配置文件 -->
  <include file="$(find bitathome_navigation_control)/launch/tb_amcl.launch" />

  <!-- 导入who is who的move_bsae的配置文件 -->
  <include file="$(find bitathome_navigation_control)/launch/wiw_move_base.launch" />

  <node pkg="bitathome_navigation_control" name="my_move_base" type="my_move_base.py" output="screen" />

  <node pkg="tf" type="static_transform_publisher" name="map_odom_broadcaster" args="0 0 0 0 0 0 /map /odom 100" />

  <!-- 打开语音节点 输出所说内容 -->
  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" output="screen" />
  <node name="AudioPlay" pkg="bitathome_remote_control" type="AudioPlay.py" output="screen" />
</launch>
